gapic/src/main/com/google/gapid/server/GapisProcess.java

```diff
@@ -113,6 +128,10 @@ protected Exception prepare(ProcessBuilder pb) {
       args.add(adb);
     }
 
+    if (gapisArgs.isSpecified()) {
+      args.add(gapisArgs.get());
```

Comment:
The help text says "argument**s**", however this passes the string as a single argument.